packages/components/src/Image/Image.tsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
-import { Box, system } from '@marigold/system';
+import { createStyles, system } from '@marigold/system';
 
-type ImageProps = {};
+type ImageProps = {
+  variant?: string;
+};
+
+const useStyles = createStyles('content');
 
 export const Image = system<ImageProps, 'img'>(
-  ({ variant = 'images', children, ...props }) => {
-    return <Box as="img" themeSection="content" variant={variant} {...props} />;
+  ({ variant = 'images', ...props }) => {
+    const classNames = useStyles({ variant });
+
+    return <img className={classNames} {...props} />;
```

Comment:
add mandatory `alt` prop with default emptystring (and add it to the docs and a test)